std/node/buffer.ts

```diff
@@ -4,13 +4,22 @@ import * as base64 from "../encoding/base64.ts";
 import { Encodings, normalizeEncoding, notImplemented } from "./_utils.ts";
 
 const notImplementedEncodings = [
-  "ascii",
   "binary",
   "latin1",
   "ucs2",
   "utf16le",
 ];
 
+function decodeAscii(buffer: Uint8Array) {
+  return Array.from(buffer).map((byte) => String.fromCharCode(byte & 0x7F)).join("");
```

Comment:
Any reason this these aren't done with `TextEncoder` / `TextDecoder` ?

Reply:
Or encoding/ascii directly if that's an issue

Reply:
AFAIK TextEncoder is UTF8 only

Reply:
What about encoding/ascii in deno's code? I'm pretty sure Deno has ASCII encoding/decoding support (I vaguely recall using it)